### PR TITLE
Privilege

### PR DESCRIPTION
**Pull request by Yuriks1**
Added roll authorisation (plus removed New Zealand)

### CODE REVIEW

**Review by ToniKaru — APPROVED** [review]
Good color change and role add!

**Review by DarkendHall — CHANGES_REQUESTED** [review]
Add role to the token.
Remove multiple empty lines, it looks a bit weird when there are multiple empty lines in a row.

Other than that, great job 👍

File: frontend/tsconfig.app.tsbuildinfo [hunk 1]
```diff
@@ -0,0 +1 @@
+{"version":"4.8.4"}
```
Comment (ToniKaru) [inline frontend/tsconfig.app.tsbuildinfo:1]: Add new line 
```suggestion
{"version":"4.8.4"}

```

**Review by ToniKaru — APPROVED** [review]
Nice!

**Review by DarkendHall — APPROVED** [review]

**Pull request merged**